src/elements/content-explorer/ContentExplorer.js

```diff
@@ -220,7 +232,10 @@ class ContentExplorer extends Component<Props, State> {
 
         this.id = uniqueid('bce_');
 
+        this.contentExplorerRef = React.createRef();
+
         this.state = {
+            columnCount: 5,
```

Comment:
Try to avoid magic numbers and instead create a meaningful constant, something like `DEFAULT_COLUMN_COUNT`